### PR TITLE
Minor changes PT-BR

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

File: lib/l10n/app_pt.arb [hunk 1]
```diff
@@ -231,7 +231,7 @@
   "bottom_action_bar_receive_invoice": "Receber via Fatura",
   "bottom_action_bar_receive_btc_address": "Receber via endereço BTC",
   "bottom_action_bar_buy_bitcoin": "Comprar Bitcoin",
-  "bottom_action_bar_warning_balance_title": "Breez requer que você mantenha ao menos {balance} de saldo.",
+  "bottom_action_bar_warning_balance_title": "Saldo mínimo exigido manter na carteira: {balance}.",
```
Comment (bitcoinuser) [inline lib/l10n/app_pt.arb:234]:
```suggestion
  "bottom_action_bar_warning_balance_title": "Saldo mínimo exigido manter na Breez: {balance}.",
```

File: lib/l10n/app_pt.arb [hunk 2]
```diff
@@ -231,7 +231,7 @@
   "bottom_action_bar_receive_invoice": "Receber via Fatura",
   "bottom_action_bar_receive_btc_address": "Receber via endereço BTC",
   "bottom_action_bar_buy_bitcoin": "Comprar Bitcoin",
-  "bottom_action_bar_warning_balance_title": "Breez requer que você mantenha ao menos {balance} de saldo.",
+  "bottom_action_bar_warning_balance_title": "Saldo mínimo exigido manter na Breez: {balance}.",
```
Comment (bitcoinuser) [inline lib/l10n/app_pt.arb:234]:
```suggestion
  "bottom_action_bar_warning_balance_title": "Saldo mínimo necessário manter na Breez: {balance}.",
```

File: lib/l10n/app_pt.arb [hunk 3]
```diff
@@ -231,7 +231,7 @@
   "bottom_action_bar_receive_invoice": "Receber via Fatura",
   "bottom_action_bar_receive_btc_address": "Receber via endereço BTC",
   "bottom_action_bar_buy_bitcoin": "Comprar Bitcoin",
-  "bottom_action_bar_warning_balance_title": "Breez requer que você mantenha ao menos {balance} de saldo.",
+  "bottom_action_bar_warning_balance_title": "Saldo mínimo necessário manter na Breez: {balance}.",
```
Comment (bitcoinuser) [inline lib/l10n/app_pt.arb:234]:
```suggestion
  "bottom_action_bar_warning_balance_title": "A Breez exige um saldo mínimo de {balance}.",
```

File: lib/l10n/app_pt.arb [hunk 4]
```diff
@@ -231,7 +231,7 @@
   "bottom_action_bar_receive_invoice": "Receber via Fatura",
   "bottom_action_bar_receive_btc_address": "Receber via endereço BTC",
   "bottom_action_bar_buy_bitcoin": "Comprar Bitcoin",
-  "bottom_action_bar_warning_balance_title": "Breez requer que você mantenha ao menos {balance} de saldo.",
+  "bottom_action_bar_warning_balance_title": "A Breez exige um saldo mínimo de {balance}.",
```
Comment (bitcoinuser) [inline lib/l10n/app_pt.arb:234]:
```suggestion
  "bottom_action_bar_warning_balance_title": "A Breez exige manter um saldo mínimo de {balance}.",
```

File: lib/l10n/app_pt.arb [hunk 5]
```diff
@@ -231,7 +231,7 @@
   "bottom_action_bar_receive_invoice": "Receber via Fatura",
   "bottom_action_bar_receive_btc_address": "Receber via endereço BTC",
   "bottom_action_bar_buy_bitcoin": "Comprar Bitcoin",
-  "bottom_action_bar_warning_balance_title": "Breez requer que você mantenha ao menos {balance} de saldo.",
+  "bottom_action_bar_warning_balance_title": "A Breez exige manter um saldo mínimo de {balance}.",
```
Comment (bitcoinuser) [inline lib/l10n/app_pt.arb:234]:
```suggestion
  "bottom_action_bar_warning_balance_title": "A Breez exige um saldo mínimo de {balance}.",
```

File: lib/l10n/app_pt.arb (hunk 4 above)
Comment (bitcoinuser) [inline lib/l10n/app_pt.arb:234]:
```suggestion
  "bottom_action_bar_warning_balance_title": "Saldo mínimo exigido pela Breez: {balance}.",
```

File: lib/l10n/app_pt.arb [hunk 6]
```diff
@@ -231,7 +231,7 @@
   "bottom_action_bar_receive_invoice": "Receber via Fatura",
   "bottom_action_bar_receive_btc_address": "Receber via endereço BTC",
   "bottom_action_bar_buy_bitcoin": "Comprar Bitcoin",
-  "bottom_action_bar_warning_balance_title": "Breez requer que você mantenha ao menos {balance} de saldo.",
+  "bottom_action_bar_warning_balance_title": "Saldo mínimo exigido pela Breez: {balance}.",
```
Comment (bitcoinuser) [inline lib/l10n/app_pt.arb:234]:
```suggestion
  "bottom_action_bar_warning_balance_title": "Breez requer que você mantenha {balance} de saldo .",
```

File: lib/l10n/app_pt.arb [hunk 7]
```diff
@@ -231,7 +231,7 @@
   "bottom_action_bar_receive_invoice": "Receber via Fatura",
   "bottom_action_bar_receive_btc_address": "Receber via endereço BTC",
   "bottom_action_bar_buy_bitcoin": "Comprar Bitcoin",
-  "bottom_action_bar_warning_balance_title": "Breez requer que você mantenha ao menos {balance} de saldo.",
+  "bottom_action_bar_warning_balance_title": "Breez requer que você mantenha {balance} de saldo .",
```
Comment (bitcoinuser) [inline lib/l10n/app_pt.arb:234]:
```suggestion
  "bottom_action_bar_warning_balance_title": "Breez requer que você guarde ao menos {balance} no saldo .",
```

File: lib/l10n/app_pt.arb [hunk 8]
```diff
@@ -231,7 +231,7 @@
   "bottom_action_bar_receive_invoice": "Receber via Fatura",
   "bottom_action_bar_receive_btc_address": "Receber via endereço BTC",
   "bottom_action_bar_buy_bitcoin": "Comprar Bitcoin",
-  "bottom_action_bar_warning_balance_title": "Breez requer que você mantenha ao menos {balance} de saldo.",
+  "bottom_action_bar_warning_balance_title": "Breez requer que você guarde ao menos {balance} no saldo .",
```
Comment (bitcoinuser) [inline lib/l10n/app_pt.arb:234]:
```suggestion
  "bottom_action_bar_warning_balance_title": "Breez requer que você possua ao menos {balance} no saldo .",
```

File: lib/l10n/app_pt.arb [hunk 9]
```diff
@@ -231,7 +231,7 @@
   "bottom_action_bar_receive_invoice": "Receber via Fatura",
   "bottom_action_bar_receive_btc_address": "Receber via endereço BTC",
   "bottom_action_bar_buy_bitcoin": "Comprar Bitcoin",
-  "bottom_action_bar_warning_balance_title": "Breez requer que você mantenha ao menos {balance} de saldo.",
+  "bottom_action_bar_warning_balance_title": "Breez requer que você possua ao menos {balance} no saldo .",
```
Comment (bitcoinuser) [inline lib/l10n/app_pt.arb:234]:
```suggestion
  "bottom_action_bar_warning_balance_title": "Breez requer que você mantenha ao menos {balance} no saldo .",
```

File: lib/l10n/app_pt.arb [hunk 10]
```diff
@@ -231,7 +231,7 @@
   "bottom_action_bar_receive_invoice": "Receber via Fatura",
   "bottom_action_bar_receive_btc_address": "Receber via endereço BTC",
   "bottom_action_bar_buy_bitcoin": "Comprar Bitcoin",
-  "bottom_action_bar_warning_balance_title": "Breez requer que você mantenha ao menos {balance} de saldo.",
+  "bottom_action_bar_warning_balance_title": "Breez requer que você mantenha ao menos {balance} no saldo .",
```
Comment (bitcoinuser) [inline lib/l10n/app_pt.arb:234]:
```suggestion
  "bottom_action_bar_warning_balance_title": "Breez requer que você mantenha ao menos {balance} no saldo.",
```

**Comment by ademar111190** [comment]
Thank you very much @bitcoinuser 🙏

**Pull request merged**